### ISSUE TITLE
Filter lists don't work with uBlock Origin v1.17.7b8

### ISSUE THREAD

**Issue by Grossdm** [DandelionSprout/adfilt#23]
Hello,
The following lists have problems with uBO:
`https://raw.githubusercontent.com/DandelionSprout/adfilt/master/stayingonbrowser/Staying%20On%20The%20Phone%20Browser`
This used to display the list's name (and I think load the list correctly), but it now fails to load the list and its name.

`https://raw.githubusercontent.com/DandelionSprout/adfilt/master/StopAutoplayOnYouTube.txt`
I just tried to add this list. It appears to load, but the list's name does not display.

![filterlistwoes](https://user-images.githubusercontent.com/836787/51368632-52657500-1abe-11e9-99c8-09cd1d789d6a.jpg)

Thank you for making your lists available!
Doug

**Comment by DandelionSprout** [comment]
Hello. 🙂 

I believe I've found the problems. On *Stop Autoplay on YouTube*, I had managed to use `! Name:` instead of the correct `! Title:`, which has been fixed now.

On *Staying on the Phone Browser*, I had recently split off some entries into an extension file; but due to the convoluted and unclear rules that uBO have placed on the use of `!#include`, the extension file was in a folder tree position where uBO refused to load from. It has now been moved to a better position, and should now be working again.

I don't tend to get a lot of feedback on my international, so that's why I were previously unaware of these problems, and also makes me appreciate feedback like yours so much more. 👍
